### ISSUE TITLE
Online client, unexpected SC2215 error that should've been fixed by #430

### ISSUE THREAD

**Issue by JWCS** [hadolint/hadolint#980]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

Note: there is an existing closed issue, #347, where it was supposed to be solved (in #430), but I'm uncertain if this is an out of date build for the online client, or such.

### Expected behavior
No SC2215 error.
### Actual behavior
SC2215 error on RUN commands with a --mount (dashed option) as the first argument.

### Steps to reproduce the behavior
This is run in the online linter.

**Dockerfile**

```Dockerfile
# syntax=docker/dockerfile:experimental
# ^ included because of #347 , no changes
FROM ubuntu:2204
# Error: SC2215
RUN --mount=type=cache,target=/var/cache/apt \
    apt-get update && apt-get install software-properties-common -y --no-install-recommends \
    && rm -rf /var/lib/apt/lists/*
```

**Comment by miigotu** [comment]
```dockerfile


RUN --security=insecure curl --proto "=https" --tlsv1.2 -sSf https://sh.rustup.rs | sed "s#/proc/self/exe#$SHELL#g" | sh -s -- -y --profile minimal --default-toolchain nightly

**Comment by xeniape** [comment]
Are there any updates on this issue?

**Comment by m-ildefons** [comment]
Online linter has been updated to v2.14.0, where this is fixed. Closing this issue.

<img width="1931" height="1534" alt="Image" src="https://github.com/user-attachments/assets/cd5b1586-3d4c-43ed-8969-c6f7e769d054" />
